README.md

```diff
@@ -35,5 +35,31 @@ DATABASE_URL=postgres://jrob@localhost:5432/floworky
 * Test Spies, Stubs and Mocks: [Sinon](http://sinonjs.org/)
 * Test Coverage: [Istanbul](https://github.com/gotwarlost/istanbul)
 
+### Flow
+In order to configure flow with the project run the ```npm install``` command within the project directory in the terminal. Flow is already configured within the package.json and also in the .flowconfig file, which directs flow to ignore the node_modules and to use the files found within the declarations directory as a source for a library of pre-determined types based on the sequelize object models in the database folder. Then, use the flow pragma at the beginning of each file to specify which files you would want flow to type check. In other words, on line 1 of the file, write ```// @flow``` or. If you want a less strict test with less errors use the ```//@flow weak``` pragma before using the first one. Once you have put the flow pragma in the files that you want to check, use the npm script```npm run flow``` to check the files. You can also configure flow to run inside of your editor by using the Nuclide IDE.
```

Comment:
Can we wordsmith this a little to make it more concise?  I can help with this, just feels like a big wall of text to me right now, with some code/instructions hidden in the middle.